django_filters/filters.py

```diff
@@ -235,6 +240,12 @@ def filter(self, qs, value):
 
         return qs.distinct() if self.distinct else qs
 
+    def get_filter_predicate(self, v):
+        try:
+            return {self.name: getattr(v, self.field.to_field_name)}
+        except AttributeError:
+            return {self.name: v}
```

Comment:
My thought is to go with this. If someone wants to profile it etc, I'd look at it then...

If it really bothers with you I'm happy for you to change it. 

But, as I said, I'd go with this first pass.